mono/metadata/handle.c

```diff
@@ -245,9 +294,9 @@ mono_handle_new_interior (gpointer rawptr, const char *owner)
 HandleStack*
 mono_handle_stack_alloc (void)
 {
-	HandleStack *stack = g_new0 (HandleStack, 1);
-	HandleChunk *chunk = g_new0 (HandleChunk, 1);
-	HandleChunk *interior = g_new0 (HandleChunk, 1);
+	HandleStack *stack = new_handle_stack ();
+	HandleChunk *chunk = new_handle_chunk ();
+	HandleChunk *interior = new_handle_chunk ();
```

Comment:
Oops... just realized that this isn't okay actually... we were relying on `g_new0` to initialize `chunk->size`, `chunk->next`, `chunk->prev` and `interior->size` to 0.